src/App.tsx

```diff
@@ -1,16 +1,27 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [moviesList, setMoviesList] = useState(moviesFromServer);
+  const onAdd = (newMovie: Movie) => {
```

Comment:
```suggestion
  const addMovie = (newMovie: Movie) => {
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/de17c3f4-ce42-4764-8745-68255b69dab8)